Gaijinizer patch/CommonEvents_117.txt

```diff
@@ -5,74 +5,79 @@
 が…あぁ……♡♡
 > CONTEXT: CommonEvents_117/3/Dialogue/kazuya-0
 > CONTEXT: CommonEvents_121/3/Dialogue/kazuya-2
-Oh...♡. There...Ah...... ♡♡#MTLed
+Aah...♡
+
+Ghh... aaah... ♡♡#Ron
 > END STRING
 
 > BEGIN STRING
 あーあ、数秒ももたなかったねー♪
 
 思ったより早かったなぁ…ちょっとは耐えるかと思ったのに。
 > CONTEXT: CommonEvents_117/7/Dialogue/enemy2
-Ah, ah, I didn't even have a few seconds♪
-It was faster than I thought...I thought I
-would endure for a moment.#MTLed
+Oh no, you couldn't hold it even a few seconds~. ♪
+That was even faster than I thought... I thought
+you'd be able to hold it in a little longer than
+that.#Ron
```

Comment:
I'd replace the second "thought" with "expected" so we don't repeat the same word too many times in a row.